### PR TITLE
[RFC] Started documenting custom backend routes.

### PR DESCRIPTION
**Pull request by sheeep**
I've have not documented a feature before, so this might be out of shape and/or place. Can I get an opinion @aschempp 🙂?

### CODE REVIEW

File: cookbook/customizing-contao/custom-backend-routes.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+```yaml
+# app/config/routing.yml
+
+app:
+    resource: '@AppBundle/Controller/'
+    type:     annotation
+```
+
+Our route `backendRouteAction` will render the template `AppBuundle::my_backend_route.html.twig`
```
Comment (fiedsch) [inline cookbook/customizing-contao/custom-backend-routes.md]: `AppBuundle` -> `AppBundle`

File: cookbook/customizing-contao/custom-backend-routes.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+{% block main %}
+    <div class="tl_listing_container">
+        Main Content.
+    </div>
+{% endblock %}
+```
+
+As we extend from `@ContaoCore/Backend/be_page.html.twig` it is worth noting 
+that there are three different block you can currently use:
```
Comment (fiedsch) [inline cookbook/customizing-contao/custom-backend-routes.md]: block**s**

**Comment by sheeep** [comment]
Thx @fiedsch, I fixed everything you mentioned.

**Comment by aschempp** [comment]
using the cookbook looks good to me.

**Comment by koertho** [comment]
Thanks @sheeep for this great tutorial. Didn't knew about this possiblity yet. Maybe there should be a note, that this is Contao 4 only.

**Pull request closed without merging**